backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -54,6 +54,7 @@ describe('UserResolver', () => {
       lastName: 'Lustig',
       language: 'de',
       publisherId: 1234,
+      redeemCode: 'a0000b0000c0000',
```

Comment:
Id do not like this here. The redeem code is an optional argument. If provided, the resolver must trigger some more actions. So if you provide it, then test these actions, which is not the case.

Reply:
ok